src/Entity/ListBuilder/DeveloperAppListBuilderForDeveloper.php

```diff
@@ -215,4 +216,21 @@ public function pageTitle(): TranslatableMarkup {
     return $title;
   }
 
+  /**
+   * Returns app credentials.
+   *
+   * @return \Symfony\Component\HttpFoundation\JsonResponse
+   *   The app credentials.
+   */
+  public function appCredentials($developer_app_by_name): JsonResponse {
+    $payload = [];
+    if ($credentials = array_shift($developer_app_by_name->getCredentials())) {
```

Comment:
The Apigee Edge module supports app with multiple credentials, this feature should not be removed with this change.

Reply:
@arlina-espinoza do you have an example of the app with multiple credentials ?

Reply:
Go to Apigee Edge MGMT UI, can you can add more than one certificate to an app on the UI, those should be visible in the devportal too even if they cannot be created on the devportal UI by default.

Reply:
Yes, found it and made the change to support multiple credentials.